doc/source/singlevisit.rst

```diff
@@ -0,0 +1,544 @@
+the exposures
+
+.. list-table:: Processing keywords 
+  :widths: 10 15 40
+  :header-rows: 1
+  
+  * - Header Keyword
+    - Valid Values
+    - Notes
+  * - FLATCORR
+    - COMPLETED
+    - Completion of basic calibration
+  * - DRIZCORR
+    - COMPLETED
+    - Completion of distortion calibration
+  * - WCSNAME
+    - \-FIT
+    - Successful **a posteriori** alignment
+  * -
+    - \-HSC30
+    - Successful **a priori** alignment
+  * -
+    - \-GSC240
+    - Successful **a priori** alignment
+
+The full set of possibilities for updated WCSs as reported using the **WCSNAME**
+keyword can be found in the description of the :ref:`wcsname-conventions`.
+
+As long as the input data meets these requirements, then SVM processing will have
+the best chance of success.  Data which has not been able to be aligned successfully
+with an **a priori** or **a posteriori** solution can still be processed as part
+of a single-visit, however, the alignment may be more difficult to determine due 
+to the larger uncertainties for HST pointing prior to October 2017.  
+
+
+Filtering the input data
+--------------------------
+Not all HST imaging observations can be aligned using SVM processing.  Observations
+taken with the GRISM or in SPATIAL SCAN mode result in sources which can not be 
+aligned, for example.  The :ref:`analyze_api` module evaluates all
+input exposures using these header keywords for the stated rejection criteria.
+
+.. list-table:: Single-visit product filenames
+  :widths: 26 27 60
+  :header-rows: 1
+  
+  * - Header Keyword
+    - Values Which Trigger Rejection
+    - Explanation
+  * - OBSTYPE
+    - (not IMAGING)
+    - Only Imaging mode data processed
+  * - MTFLAG
+    - T 
+    - No moving targets, WCS and background sources vary
+  * - SCAN_TYP
+    - C or D (or not N)
+    - Can not align streaked sources
+  * - FILTER or FILTER1, FILTER2
+    - C*, PR*
+    - G=Grism and PR=Prism, Can not align streaked sources
+  * - EXPTIME
+    - 0 
+    - no exposure time, no data to align
+  * - TARGNAME
+    - DARK, TUNGSTEN, BIAS, FLAT, 
+    - No alignable external sources in these calibration modes 
+  * - 
+    - EARTH-CALIB, DEUTERIUM
+    - No alignable external sources in these calibration modes 
+  * - CHINJECT
+    - not NONE
+    - No alignable external sources in these calibration modes 
+
+
+Any observation which meets any of these criteria are flagged to be ignored (not
+processed).  All observations which are alignable based on these criteria are then
+passed along as a table to create the SVM products.  Those inputs which can be
+processed are then copied and renamed using the :ref:`svm_naming_convention`.  This 
+insures that no SVM processing will affect or otherwise modify the original 
+pipeline-processed input files.  Only the SVM named input files will be updated
+with new SVM-aligned WCS solutions and then used to produce the drizzle products.  
+
```

Comment:
It looks like the the FGSLOCK contains 'COARSE' or 'GY', the data will be processed for alignment, but the results could be compromised.